### ISSUE TITLE
Suggestion: Wardrobe

### ISSUE THREAD

**Issue by BernisonCorw** [133spider/AQLite#52]
Hello,

I'm an admirer of your project and I really respect the effort and dedication you're puting to it. I'm not a programer so this suggestion is hipotetical, I hope it's possible to implement in your project "AQLite".

Wardrobe option is an option that will allow you to save the combination of items you currently have on, and name the "set" you've saved.
Therefore there will be a list of saved sets that you can put on by just clicking on them and clicking on "Equip" so you can instantly put on multpiple items at a time even if the items are in your bank (Ofcourse you must have possesion of those items in your bank or backpack). This list can be organized by "groups" named by the player itself. 
In that list you will find all the names of the sets you've saved and by clicking on them you'll be able to preview the set. 

It can be placed in "caracter". [Image 1]
![Image 1](https://user-images.githubusercontent.com/66405593/83746541-13cd4d80-a614-11ea-848b-c59c3ac20f04.PNG)

Idk if it's possible to set it up the way I'm thinking bbbut... the main idea is to put a wardrobe for sets in AQLite and the option to equip many items at the same time with one click. Items from bank and Inventory. :)

**Comment by 133spider** [comment]
Great idea on having it on that UI icon!
This idea has been floating around on my to-do list, but the main issue that I have to solve is figuring out how to equip more than one item in a rapid and smooth succession. In the game, equipping items have a cooldown before you can equip another item. I'll continue to think up alternate solutions for this feature.

**Comment by BernisonCorw** [comment]
Yeah, I think equiping items that are inside the bank will be a real challenge and it wont be possible. So It's fine if you're just able to setup multi-equipment for inventory only.
